common/flavorization/wc_flavorization.txt

```diff
@@ -0,0 +1,39 @@
+﻿#Princes (children of Kings/Emperors, sorted by culture)
+
+### Dragons ###
+prince_male_dragon_kingdom {
```

Comment:
Other titles in this file have the same problem.
```suggestion
prince_male_dragon_kingdom = {
```

Reply:
fixed in f00bdd298